samples/algorithms/sudoku-grover/Program.cs

```diff
@@ -0,0 +1,385 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Quantum.Simulation.Core;
+using Microsoft.Quantum.Simulation.Simulators;
+
+namespace Microsoft.Quantum.Samples.SudokuGrover
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var sim = new QuantumSimulator(throwOnReleasingQubitsNotInZeroState: true);
+
+            // Test solving a 4x4 Sudoku puzzle using classical computing
+            // missing numbers are denoted by 0
+            int[,] puzzle4 = {
```

Comment:
Would it be an idea to add an input argument to the program that lets you pick a puzzle to run instead of running all puzzles at once? It can get a bit overwhelming to read all these results in stdout.

Reply:
done.  Add arg as follows to choose that option
-  all or blank : run all puzzles (default)
-  4x4-classic : test classic algorthm on a 4x4 puzzle
-  4x4-1 : test Quantum solution of 4x4 puzzle missing 1 number
-  4x4-3 : test Quantum solution of 4x4 puzzle missing 3 numbers
-  4x4-4 : test Quantum solution of 4x4 puzzle missing 4 numbers
-  9x9-1 : test classic algorithm and Quantum solution on a 
            9x9 puzzle with 1 missing number
-  9x9-2 : test Quantum solution on a 
            9x9 puzzle with 2 missing numbers
- 9x9-64   : test classic algorithm and Quantum solution on a 
             9x9 puzzle with 64 missing numbers